docs/API/LorisRESTAPI.md

```diff
@@ -696,3 +696,76 @@ The JSON object is of the form:
     "Value" : string
 }
 ```
+
+# 5.0 DICOM Data
+
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
+portion of the REST API namespaces, but is defined in a separate section of this 
+document for clarity purposes.
+
+## 5.1 Candidate DICOMs
+```
+GET /candidates/$CandID/$Visit/dicoms
+```
+
+A GET request to `/candidates/$CandID/$Visit/dicoms` will return a JSON object of
+all the images which have been acquired for that visit. It will return an object of
+the form:
+
```

Comment:
I think this should be "one of the following forms", either that or there's something missing here.

Reply:
the before 17.1 and starting 17.2 release statements were not intended to be kept at all; so removed those (now we have only one form).